packages/synapse-react-client/src/components/StandaloneQueryWrapper/StandaloneQueryWrapper.tsx

```diff
@@ -46,6 +46,7 @@ type StandaloneQueryWrapperOwnProps = {
     | 'columnAliases'
     | 'noContentPlaceholderType'
     | 'showLastUpdatedOn'
+    | 'visibleColumnCount'
```

Comment:
This was missing in the prop signature, but the prop was still passed down, so this is a type-only change.